tests/login.test.ts

```diff
@@ -0,0 +1,61 @@
+import { test, expect } from '@playwright/test';
+import { HomePage } from '../pages/home.page';
+import { LoginPage } from '../pages/login.page';
+import { MyAccountPage } from '../pages/myAccount.page';
+import { HeaderComponent } from '../components/header.components';
+import { userCredentials } from '../test-data/users.data';
+
+test.describe('Login tests', () => {
+    let loginPage: any;
+
+    test.beforeEach(async ({ page }) => {
+        const homePage = new HomePage(page);
+        loginPage = new LoginPage(page);
+        const header = new HeaderComponent(page);
+
+        await homePage.goToHomePage();
+        await header.clickSignInButton();
+    });
+
+    test('Successful login with valid credentials', async ({ page }) => {
+        loginPage = new LoginPage(page);
+        const myAccount = new MyAccountPage(page);
+
+        await loginPage.singInUser(userCredentials.userEmail, userCredentials.userPassword);
```

Comment:
const { userEmail,  userPassword } = userCredentials; add before test.describe and use only those declarations

Reply:
Solved.